### ISSUE TITLE
Bug in utimers

### ISSUE THREAD

**Issue by ZarTek-Creole** [eggheads/eggdrop#1655]
With timers:

```
<@ZarTek> tcl return [timer ${::bonaPRE::mysql_(conrefresh)} [list ::bonaPRE::MySQL::KeepAlive] 1 ZarTest]
<bona11> Return: ZarTest - 0.022 ms
<@ZarTek> tcl return [timer ${::bonaPRE::mysql_(conrefresh)} [list ::bonaPRE::MySQL::KeepAlive] 1 ZarTest]
<bona11> Error: timer already exists by that name - 0.033 ms
<@ZarTek> tcl return [timers]
<bona11> Return: {595 ::bonaPRE::MySQL::KeepAlive ZarTest 1} - 0.017 ms
```

with utimers:

```
<@ZarTek> tcl return [utimer ${::bonaPRE::mysql_(conrefresh)} [list ::bonaPRE::MySQL::KeepAlive] 1 SQLKeepAlive]
<bona11> Return: SQLKeepAlive - 0.042 ms
<@ZarTek> tcl return [utimer ${::bonaPRE::mysql_(conrefresh)} [list ::bonaPRE::MySQL::KeepAlive] 1 SQLKeepAlive]
<bona11> Return: SQLKeepAlive - 0.053 ms
<@ZarTek> tcl return [utimer ${::bonaPRE::mysql_(conrefresh)} [list ::bonaPRE::MySQL::KeepAlive] 1 SQLKeepAlive]
<bona11> Return: SQLKeepAlive - 0.023 ms
<@ZarTek> tcl return [utimers]
<bona11> Return: {267 ::bonaPRE::MySQL::KeepAlive SQLKeepAlive 1} {475 ::bonaPRE::MySQL::KeepAlive SQLKeepAlive 1} {489 ::bonaPRE::MySQL::KeepAlive SQLKeepAlive 1} {591 ::bonaPRE::MySQL::KeepAlive SQLKeepAlive 1} {593 ::bonaPRE::MySQL::KeepAlive SQLKeepAlive 1} {595 ::bonaPRE::MySQL::KeepAlive SQLKeepAlive 1} - 0.033 ms
```

**Comment by vanosg** [comment]
In a sentence, can you summarize the issue you're seeing?

**Comment by thommey** [comment]
Timer IDs should not be allowed to duplicate, but right now that check is broken for utimers because it checks the timer-list instead of the utimer-list.

Expected:
```
.tcl timer 1 testproc 1 testid
Tcl:
.tcl timer 1 testproc 1 testid
Tcl error: timer already exists by that name

.tcl utimer 1 testproc 1 testid2
Tcl:
.tcl utimer 1 testproc 1 testid2
Tcl error: timer already exists by that name
```

Only the first part works, utimer duplicate check does not work. After #1656 it works.
